openmc/cell.py

```diff
@@ -197,7 +196,14 @@ def rotation(self):
 
     @property
     def rotation_matrix(self):
-        return self._rotation_matrix
+        if self.rotation is not None:
```

Comment:
I'm just curious if you think it *might* be worth saving the rotation matrix. If one were to iterate and query the geometry for the cell for a bunch of locations, it might be nice to have the matrix cached. I would have thought that Python plotting would be an example of this use case, but perhaps not.